test/APITests.js

```diff
@@ -309,6 +310,64 @@ describe('API Tests', () => {
       });
     });
 
+    it('has option getter', () => {
+      [div, api] = createElementAndApi(
+        simpleCenterViewConfig, { editable: false, bounded: true }
+      );
+
+      expect(api.option('editable')).toEqual(false);
+      expect(api.option('bounded')).toEqual(true);
+      expect(api.option('scrollable')).toEqual(undefined);
+    });
+
+    it('can scroll when scrollable is true', (done) => {
+      [div, api] = createElementAndApi(
+        stackedTopTracks,
+        { editable: false, scrollable: true },
+        600, 200, true
+      );
+
+      expect(api.option('scrollable')).toEqual(true);
+
+      const hgc = api.getComponent();
+
+      waitForTilesLoaded(hgc, () => {
+        const trackRenderer = document.querySelector('.track-renderer-events');
+        trackRenderer.scrollTop = 20;
+
+        setTimeout(() => {
```

Comment:
Why setTimeout? Is this so that the even queue has a chance to process all pending events before you check for the correct values?

Reply:
Good catch. I previously used an event to trigger scrolling until I realized that's not how the scroll event works. But then I forgot to remove the timeout again. I just removed it.